Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -894,32 +894,32 @@
   "block.biomeswevegone.white_allium": "Белый лук",
   "block.biomeswevegone.white_anemone": "Белая ветреница",
-  "block.biomeswevegone.white_mangrove_bookshelf": "Книжная полка из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_button": "Кнопка из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_crafting_table": "Верстак из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_door": "Дверь из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_fence": "Забор из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_fence_gate": "Калитка из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_hanging_sign": "Подвесная табличка из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_leaves": "Листва белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_log": "Древесина белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_planks": "Доски из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_pressure_plate": "Нажимная плита из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_sapling": "Саженец белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_sign": "Табличка из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_slab": "Плита из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_stairs": "Ступени из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_trapdoor": "Люк из белого мангрового дерева",
-  "block.biomeswevegone.white_mangrove_wood": "Кора белого мангрового дерева",
+  "block.biomeswevegone.white_mangrove_bookshelf": "Книжная полка из белой мангры",
+  "block.biomeswevegone.white_mangrove_button": "Кнопка из белой мангры",
+  "block.biomeswevegone.white_mangrove_crafting_table": "Верстак из белой мангры",
+  "block.biomeswevegone.white_mangrove_door": "Дверь из белой мангры",
+  "block.biomeswevegone.white_mangrove_fence": "Забор из белой мангры",
+  "block.biomeswevegone.white_mangrove_fence_gate": "Калитка из белой мангры",
+  "block.biomeswevegone.white_mangrove_hanging_sign": "Подвесная табличка из белой мангры",
+  "block.biomeswevegone.white_mangrove_leaves": "Листья белой мангры",
+  "block.biomeswevegone.white_mangrove_log": "Древесина белой мангры",
+  "block.biomeswevegone.white_mangrove_planks": "Доски из белой мангры",
+  "block.biomeswevegone.white_mangrove_pressure_plate": "Нажимная плита из белой мангры",
+  "block.biomeswevegone.white_mangrove_sapling": "Саженец белой мангры",
+  "block.biomeswevegone.white_mangrove_sign": "Табличка из белой мангры",
+  "block.biomeswevegone.white_mangrove_slab": "Плита из белой мангры",
+  "block.biomeswevegone.white_mangrove_stairs": "Ступеньки из белой мангры",
+  "block.biomeswevegone.white_mangrove_trapdoor": "Люк из белой мангры",
+  "block.biomeswevegone.white_mangrove_wood": "Белая мангра",
```

Я имел в виду "Белый мангр"
```suggestion
  "block.biomeswevegone.white_mangrove_wood": "Белый мангр",
```
Тут легко исправить заменой.